backend/app/common.py

```diff
@@ -1,9 +1,26 @@
 """
 Miscellaneous utility functions useful throughout the system
 """
+import base64
+import io
+from scipy.io import wavfile
 from textwrap import dedent
 
 
+def wav_to_base64(byte_array, sample_rate):
+    """
+    Encode the WAV byte array with base64
+    :param byte_array: int16 numpy array
+    :param sample_rate: integer, the sampling rate
+    :return: base64 encoding of the given array as a str
+    """
+    byte_io = io.BytesIO(bytes())
+    wavfile.write(byte_io, sample_rate, byte_array)
+    wav_bytes = byte_io.read()
+    audio_data = base64.b64encode(wav_bytes).decode('UTF-8')
+    return audio_data
```

Comment:
I'd be super interested to see if we could write a unit test for this. We don't need to use the actual sample rate we'll use for the rest of the product (44100), so could you construct a byte array with a significantly lower sample rate in a test, pass it through this function, and assert that the return value is what we expect? In addition to the kind of regression testing that'll provide us for the future, providing a working example of how this function operates in a test will help our future selves understand how to use this function better.